School.js

```diff
@@ -8,60 +8,60 @@ class School {
       //   physics: {} 
     }
   }
-
-  /**
-   * Add class to classes
-   * 
-   * @param {string} name - Name of the class
-   * @param {string} teacher - Name of instructor 
-   * @return {Class} Class object
-   */
+  // /**
+  //  * Add class to classes
+  //  * 
+  //  * @param {string} name - Name of the class
+  //  * @param {string} teacher - Name of instructor 
+  //  * @return {Class} Class object
+  //  */
   addClass(name, teacher) {
     let newClass = new Class(name, teacher);
     this.classes[name] = newClass;
   }
 
-  /**
-   * Enroll student in class
-   * 
-   * @param {string} className - Name of the class
-   * @param {Student} student - Student object
-   * @return {Student} Enrolled student
-   */
+  // /**
+  //  * Enroll student in class
+  //  * 
+  //  * @param {string} className - Name of the class
+  //  * @param {Student} student - Student object
+  //  * @return {Student} Enrolled student
+  //  */
   enrollStudent(className, student) {
-    // Your code here
-  }
-
-
-
+    let enrollClass = this.classes[className].students;
+    let newStudent = new Student(student.name, student.age, student.city, student.grade);
+    enrollClass.push(newStudent)
+    }
```

Comment:
You need to return your `newStudent` here so you could use it to include in your response